ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -341,37 +345,21 @@ export default class TaskRunsCard extends Component {
                       </ListItem>
                     </List>
                   </Collapse>
-                  <CopyToClipboard
-                    title={`${run.scheduled} (Copy)`}
-                    text={run.scheduled}>
-                    <ListItem button className={classes.listItemButton}>
-                      <ListItemText
-                        primary="Scheduled"
-                        secondary={<DateDistance from={run.scheduled} />}
-                      />
-                      <ContentCopyIcon />
-                    </ListItem>
-                  </CopyToClipboard>
-                  <CopyToClipboard
-                    title={`${run.started} (Copy)`}
-                    text={run.started}>
-                    <ListItem button className={classes.listItemButton}>
-                      <ListItemText
-                        primary="Started"
-                        secondary={
-                          run.started ? (
-                            <DateDistance
-                              from={run.started}
-                              offset={run.scheduled}
-                            />
-                          ) : (
-                            <em>n/a</em>
-                          )
-                        }
-                      />
-                      <ContentCopyIcon />
-                    </ListItem>
-                  </CopyToClipboard>
+                  <ListItem>
+                    <ListItemText
+                      primary="Reason Resolved"
+                      secondary={
+                        run.reasonResolved ? (
+                          <StatusLabel
+                            state={run.reasonResolved}
+                            className={classes.reasonResolved}
```

Comment:
The grey coloring should also be done for "Reason Created".

Reply:
@djmitche Done